beesdoo_shift_swap/views/res_config_setting_view.xml

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<odoo>
+    <data>
+
+        <record id="res_config_settings_swap_view_form" model="ir.ui.view">
+            <field name="name">res.config.settings.view.form</field>
+            <field name="model">res.config.settings</field>
+            <field name="priority" eval="50"/>
+            <field name="inherit_id" ref="base.res_config_settings_view_form"/>
+            <field name="arch" type="xml">
+                <xpath expr="///div[hasclass('settings')]" position="inside">
```

Comment:
Did you know difference between `//` and `///` ?

Reply:
A regarder + en détail